src/theory/arith/arith_rewriter.cpp

```diff
@@ -847,6 +819,37 @@ RewriteResponse ArithRewriter::rewriteIntsDivMod(TNode t, bool pre)
   return RewriteResponse(REWRITE_DONE, t);
 }
 
+RewriteResponse ArithRewriter::rewriteExtIntegerOp(TNode t, bool pre)
```

Comment:
What is the `pre` argument for?

Reply:
Its not really necessary, I was following style of other methods but I've deleted the argument now.